cli/cycpp.py

```diff
@@ -89,13 +89,16 @@
               'boost::uuids::uuid', }
 
 BUFFERS = {'{0}::toolkit::ResourceBuff'.format(CYCNS),
+           '{0}::toolkit::ResBuf'.format(CYCNS),
            ('{0}::toolkit::ResBuf'.format(CYCNS), CYCNS + '::Resource'),
            ('{0}::toolkit::ResBuf'.format(CYCNS), CYCNS + '::Product'),
            ('{0}::toolkit::ResBuf'.format(CYCNS), CYCNS + '::Material'),
+           '{0}::toolkit::ResMap'.format(CYCNS),
            }
 
 TEMPLATES = {'std::vector', 'std::set', 'std::list', 'std::pair',
-             'std::map', '{0}::toolkit::ResBuf'.format(CYCNS),}
+             'std::map', '{0}::toolkit::ResBuf'.format(CYCNS),
+             CYCNS + '::toolkit::ResMap',}
```

Comment:
Just a style comment, dove-tailing off @jdangerx. @rwcarlsen seems to have
the most skin in this game, perhaps he should do a final review and merge.

On Sat, Jan 24, 2015 at 12:56 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In cli/cycpp.py
> <https://github.com/cyclus/cyclus/pull/1066#discussion_r23498478>:
>
> >             }
> >
> >  TEMPLATES = {'std::vector', 'std::set', 'std::list', 'std::pair',
> > -             'std::map', '{0}::toolkit::ResBuf'.format(CYCNS),}
> > +             'std::map', '{0}::toolkit::ResBuf'.format(CYCNS),
> > +             CYCNS + '::toolkit::ResMap',}
>
> Is this blocking?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1066/files#r23498478>.
>



-- 
Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192